storefront/app/helpers/spree/page_helper.rb

```
@@ -52,7 +52,7 @@ def render_section(section, variables = {}, lazy_allowed: true)
 
         path = section.lazy_path(variables)
 
-        turbo_frame_tag(css_id, src: path, loading: :lazy, class: css_class, data: { controller: 'prefetch-lazy', turbo_permanent: true }) do
+        turbo_frame_tag(css_id, src: path, loading: section.loading_strategy, class: css_class, data: { controller: 'prefetch-lazy', turbo_permanent: true }) do
```

RuboCop (1.81.7)

[convention] Line is too long. [160/150]

(Layout/LineLength)